### PR TITLE
adding bpi interop directory

### PR DESCRIPTION
**Pull request by fleischr**
# Description

Adds a folder for BPI inter

## Related Issue

#691 

## Motivation and Context

Organize and coordinate for BPI interop

## How Has This Been Tested

n/a

## Types of changes

adds bpi-interop directory under examples

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

**Review by Therecanbeonlyone1969 — DISMISSED** [review]
LGTM

**Review by Ybittan — DISMISSED** [review]
LGTM

**Review by Therecanbeonlyone1969 — DISMISSED** [review]
LGTM

**Review by Ybittan — APPROVED** [review]
LGTM

**Review by Therecanbeonlyone1969 — APPROVED** [review]
LGTM

File: examples/bpi-interop/schemas/r141-syncadvancepredicate.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "id": "uuid",
+    "originatorBpiSubject": "did_jwt_ACME", 
+    "createDate": "date",
+    "updateDate": "date",
+    "description": "",  //“The Bank Data Object of H(DID-JWT_Customer) from H(DID-JWT_ACME) and committed to as MerkleRoot(Bank Data Object) has the following structure: ${schema} and commits to a set of values to be synchronized for the data properties in the structure”,
```
Comment (skosito) [inline examples/bpi-interop/schemas/r141-syncadvancepredicate.json:6]: comments in json are not allowed

File: examples/bpi-interop/schemas/r141-syncadvancepredicate.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        "workstepId": "string - Id for Workstep 2 in BRI-1"
+    },
+    {
+        "briId": "string - Id for BRI-3",
+        "workgroupId": "string - Id for Workgroup 1 in BRI-3",
+        "workflowId": "string - Id for Workflow in BRI-3",
+        "workstepId": "string - Id for Workstep 3 in BRI-3"
+    }
+    ]
+    "proverSystem" : “Groth16 | Plonk”,
```
Comment (skosito) [inline examples/bpi-interop/schemas/r141-syncadvancepredicate.json:41]: this also looks invalid from json perspectie, quotes are not proper it seems

Comment (Therecanbeonlyone1969) [reply examples/bpi-interop/schemas/r141-syncadvancepredicate.json:41]: @skosito just raw notes rn ... we can put that into a better format over the next few weeks. We could use JSON-LD. We will update later

**Review by skosito — APPROVED** [review]

**Pull request merged**